docs/content/en/install/aks/_index.md

```diff
@@ -259,6 +259,64 @@ WARNING: version difference between client (1.21) and server (1.19) exceeds the
 
 With all of the infrastructure provisioned we can now focus on installing K8ssandra. This will require configuring a service account for the backup and restore service, creating a set of Helm variable overrides, and setting up AKS specific ingress configurations.
 
+### Create Backup / Restore Service Account Secrets
```

Comment:
I manually executed all the steps and validated they work as expected. Note that you will need this PR: https://github.com/k8ssandra/k8ssandra-terraform/pull/25.